lrpar/src/lib/dijkstra.rs

```diff
@@ -8,7 +8,7 @@ use indexmap::{
 /// Starting at `start_node`, return, in arbitrary order, all least-cost success nodes.
 ///
 /// * `neighbours` takes a node `n` and returns an iterator consisting of all `n`'s neighbouring
-///     nodes.
+///   nodes.
```

Comment:
Apparently this is the only clippy update, unrelated to this patch.

Seemed excessive to make a PR just for this one change.